### PR TITLE
Add karma config

### PR DESCRIPTION
**Pull request by yemel**

### CODE REVIEW

**Comment by maraoz** [comment]
does the project requiring this version of `gulp-bitcore` still need to include   "karma", "karma-chrome-launcher", "karma-detect-browsers", "karma-firefox-launcher", "karma-opera-launcher", "karma-mocha", etc?

edit: it would be awesome to have those dependencies only here now

**Comment by yemel** [comment]
@maraoz updated!

**Comment by eordano** [comment]
I'll test but I'm quite sure that the dependencies are not scanned for karma plugins

**Comment by eordano** [comment]
Tested, karma can't find these packages. Just drop #45941d4 from this PR and I'll merge

File: index.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ function startGulp(name, opts) {
   };
 
   var testkarma = shell.task([
-    './node_modules/karma/bin/karma start'
+    './node_modules/karma/bin/karma start ./node_modules/gulp-bitcore/karma.conf.js'
```
Comment (maraoz) [inline index.js]: please rename to `bitcore-build`

**Pull request merged**